goth/runner/container/yagna.py

```diff
@@ -50,6 +51,7 @@ def __init__(
         self,
         name: str,
         probe_type: Type["Probe"],
+        payment_config: PaymentConfig,
```

Comment:
Since this is a public field, please add its type definition to the class (like for other fields in this class).